shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -79,10 +79,11 @@ - (void)setSemanticsNode:(const blink::SemanticsNode*)node {
   return &_children;
 }
 
-- (void)neuter {
+- (void)dealloc {
   _bridge = nullptr;
   _children.clear();
   self.parent = nil;
```

Comment:
It is more idiomatic to release the parent instead of nilling it out via the setter. So `[_parent release]`